### PR TITLE
[New Theme] DarkMod Neo - phpMyAdmin 5.0

### PR DESCRIPTION
**Pull request by Suprethreal**
new dark theme for phpMyAdmin 5.0

Done my best to thoroughly test all aspects, but if you manage to find something let me know.

### CODE REVIEW

**Comment by ibennetch** [comment]
This looks great, thanks for doing the work on this.

I noticed a few minor things that don't quite seem to fit the color scheme, maybe this is intentional or maybe they were missed somewhere in the thousands of lines of css code :-)

- The text area on the SQL tab is white
- The warnings shown on the main page is the default pink (I see one because my blowfish_cookie setting is too short, that's an easy way to get the error area shown)
- The "Loading" dialog that's shown as I change pages is the default yellow

I love this. Thanks for the submission.

File: darkmod-neo/theme.json [hunk 1]
```diff
@@ -0,0 +1,8 @@
+{
+    "name": "darkmod-neo",
+    "version": "5.0",
+    "description": "DarkMod Neo, A Dark Theme for phpMyAdmin",
+    "author": "Suprethreal",
+    "url": "https://github.com/suprethreal/darkmod-neo/??? ",
```
Comment (williamdes) [inline darkmod-neo/theme.json:6]:
```suggestion
    "url": "https://github.com/suprethreal",
```
For now because no repository really exits ;)

**Comment by Suprethreal** [comment]
> * dialog

Thanks so much for the kind words.
Ideally I would change all the input fields, text areas and buttons, but that is more than I can commit to right now.
I have done several updates though including the following.

- error and notice dialog
- ajax "loading" notification
- several other small fixes and tweaks

I will make a new commit shortly.
(Once it's basically ok'd I can clean up the css files a bit and compress them again too.)

**Comment by Suprethreal** [comment]
New Commit.
Hope this is enough for approval.

Only thing left would be CodeMirror (SQL page input). Not sure how to override that in a good way since it's outside the themes folder in js.
Initial attempts to override in scss doesn't seem to work.

**Comment by Suprethreal** [comment]
Okay. CodeMirror now modified to match too.

That should be it. Please check.

**Comment by williamdes** [comment]
Hi @Suprethreal as you can see on this pull-request all the files where modified, their chmod changed
Can this change be reverted?

**Comment by Suprethreal** [comment]
> Hi @Suprethreal as you can see on this pull-request all the files where modified, their chmod changed
> Can this change be reverted?

Sorry for the trouble.
I checked against the default permissions, modified and pushed.
Downloaded the pull and looks to be okay.
How does that look?

**Comment by williamdes** [comment]
> > Hi @Suprethreal as you can see on this pull-request all the files where modified, their chmod changed
> > Can this change be reverted?
> 
> Sorry for the trouble.
> I checked against the default permissions, modified and pushed. (Only changed darkmod-neo)
> Downloaded the pull and looks to be okay.
> How does that look?

Well could you do that also for all the files in the repository?

**Comment by Suprethreal** [comment]
> > > Hi @Suprethreal as you can see on this pull-request all the files where modified, their chmod changed
> > > Can this change be reverted?
> > 
> > 
> > Sorry for the trouble.
> > I checked against the default permissions, modified and pushed. (Only changed darkmod-neo)
> > Downloaded the pull and looks to be okay.
> > How does that look?
> 
> Well could you do that also for all the files in the repository?

All should be as original permissions now.

**Comment by Suprethreal** [comment]
> Very good. Thanks for your contribution.
> 
> Before we can merge your pull request, you should revert the chmod changes.

It should be okay as of last comment.
I re-pushed with original permissions changes (for all), cloned and checked the changes stuck. Seems okay.
Git doesn't seem to pickup permission changes so it's doesn't show as a new commit.

Please check.

**Comment by mauriciofauth** [comment]
I squashed and merged your pull request after fixing the file permissions.
Thanks for your contribution.

**Comment by mauriciofauth** [comment]
You can always add new pull requests to make changes to the theme.

**Comment by mauriciofauth** [comment]
@ibennetch Could you please create a release for this theme? Thank you.

**Comment by Suprethreal** [comment]
Thanks for the feedback and support.

**Pull request closed without merging**